erpc.yaml

```diff
@@ -26,377 +34,126 @@ projects:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 42161
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 137
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 8453
-        failsafe:
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 10
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 1284
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 1285
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
+      allowedMethods:
+        - 'GET'
+        - 'POST'
+        - 'OPTIONS'
+
+    networkDefaults:
+      failsafe:
+        circuitBreaker: ~
+        timeout:
+          duration: 60s
+        retry:
+          maxAttempts: 3
+          delay: 500ms
+          backoffMaxDelay: 10s
+          backoffFactor: 0.3
+          jitter: 200ms
+        hedge:
+          delay: 1000ms
+          maxCount: 2
```

Comment:
Recommend to use quantile-based hedging which basically reduces lots of redundant calls: https://docs.erpc.cloud/config/failsafe#hedge-policy

Reply:
Thanks!